### PR TITLE
Inlined card fields

### PR DESCRIPTION
**Pull request by khanghoang**
Add inlined card fields

### CODE REVIEW

File: demo/button.htm [hunk 1]
```diff
@@ -1,29 +1,77 @@
-
 <div id="paypal-button"></div>
 
-<script src="https://www.paypalobjects.com/api/checkout.js"></script>
+<script src="http://localhost:9000/checkout.js"></script>
```
Comment (bluepnume) [inline demo/button.htm]: Let's add a separate `demo/card-fields.htm`

Comment (khanghoang) [reply demo/button.htm]: 👍

File: demo/card.htm [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            layout: 'vertical',
+            size:  'medium', // small | medium | large | responsive
+            shape: 'rect',   // pill | rect
+            color: 'gold'   // gold | blue | silve | black
+        },
+
+        client: {
+            sandbox:    'AWi18rxt26-hrueMoPZ0tpGEOJnNT4QkiMQst9pYgaQNAfS1FLFxkxQuiaqRBj1vV5PmgHX_jA_c1ncL',
+            stage: 'alc_client1',
+            production: 'AT6GwTLnW30DiiBgUHCZCZ-8gGxjpFemte_zdGkeWh1mpMFrPxM22shpzW6PkjhJ79j4G6eP7ElRPCAt',
```
Comment (bluepnume) [inline demo/card.htm]: I'd recommend removing this, otherwise people will inevitably use it. :)

File: src/card/index.js [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+                    });
+                }).finally(() => {
+                    this.close();
+                });
+            };
+        }
+    }
+});
+
+patchMethod(Card, 'init', ({ args: [ props, _context ], original, context }) => {
```
Comment (bluepnume) [inline src/card/index.js]: Can remove all the `patchMethod` calls

File: src/card/index.js [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+        iframe: true,
+        popup:  false
+    },
+
+    onAuthorize: {
+        type:     'function',
+        required: true,
+        once:     true,
+
+        decorate(original) : ?Function {
```
Comment (bluepnume) [inline src/card/index.js]: Can remove this entire decorate. This stuff is only relevant to the button.

File: src/card/index.js [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+    tag:  'zombo',
+    name: 'ppcard',
+
+    dimensions: {
+        height: '300px',
+        width:  '250px'
+    },
+
+    buildUrl() : string {
+        return 'http://localhost.paypal.com:8000/webapps/hermes/zombo';
```
Comment (bluepnume) [inline src/card/index.js]: Let's construct the urls in `config`

File: src/card/index.js [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+        required: false
+    },
+
+    onCancel: {
+        type:     'function',
+        required: false,
+        once:     true,
+        noop:     true,
+
+        decorate(original) : ?Function {
```
Comment (bluepnume) [inline src/card/index.js]: Can remove this entire block

File: src/card/index.js [hunk 7]
```diff
@@ -0,0 +1,209 @@
...
+        sameDomain: true
+    },
+
+    on: {
+        type:       'function',
+        required:   false,
+        sameDomain: true
+    },
+
+    accessToken: {
```
Comment (bluepnume) [inline src/card/index.js]: Are we using this? Otherwise can remove

**Comment by khanghoang** [comment]
It's party time 🎊🎉💃🕺

**Pull request merged**